sdk-base/src/main/java/com/mapbox/maps/plugin/locationcomponent/generated/LocationComponentSettingsInterface.kt

```diff
@@ -52,6 +53,11 @@ interface LocationComponentSettingsInterface {
    */
   var layerBelow: String?
 
+  /**
+   * The enum controls how the puck is oriented
+   */
+  var puckBearingSource: PuckBearingSource
```

Comment:
Could it be property of a `locationPuck`, `locationPuck.bearingSource` ?

Reply:
`LocationPuck` is the class to customs the UI, IMO it should not care about where the bearing value comes from just show the UI according to the value it received.